README.md

```diff
@@ -0,0 +1,136 @@
+[![Slack Status](http://slack.sparkpost.com/badge.svg)](http://slack.sparkpost.com)
+
+This is a SendGrid to SparkPost email template migration tool.
+
+The tool presents a simple web UI for migrating templates directly from your SendGrid account to SparkPost as well as translating SendGrid template text directly into its SparkPost equivalent.
+
+If you want to automate your migration, the tool also has an API.
+
+See below for details on [deployment on Heroku](#heroku-deployment), [manual deployment](#manual-deployment) and [API use](#the-api).
+
+## Supported Features
+- <%subject%>, <%body%>
+- Sender Info
```

Comment:
`Sender_Name, Sender_Address, Sender_City, Sender_State, Sender_Zip` not clear enough? I had put them all initially but then thought too verbose.